pkgs/by-name/en/ente-web/package.nix

```diff
@@ -35,18 +35,36 @@ stdenv.mkDerivation (finalAttrs: rec {
     nodejs
   ];
 
+  env.NEXT_PUBLIC_ENTE_ENDPOINT = "NIXPKGS_BUILD_ENTE_PUBLIC_ENDPOINT";
+
   installPhase = ''
     cp -r apps/photos/out $out
   '';
 
-  meta = {
-    description = "Web client for Ente Photos";
-    homepage = "https://ente.io/";
-    license = lib.licenses.agpl3Only;
-    maintainers = with lib.maintainers; [
-      surfaceflinger
-      pinpox
-    ];
-    platforms = lib.platforms.all;
-  };
+  passthru.withEndpoint = { publicEndpoint }:
```

Comment:
I don't think this is a good interface to configure this. I think a dedicated NixOS module would be better.

Reply:
Will revert this then to using a normal, optional override.

Reply:
Changed to override, see comment above.